src/pages/General.vue

```diff
@@ -32,6 +32,7 @@
 <script>
 import TelemetryOptIn from '@/components/TelemetryOptIn.vue';
 import UpdateStatus from '@/components/UpdateStatus.vue';
+import * as settingsDefault from '@/config/settings';
```

Comment:
```suggestion
import { defaultSettings } from '@/config/settings';
```
We don't need to import the whole thing; just the one should be enough.